packages/cli/src/lib.rs

```diff
@@ -279,6 +286,44 @@ where
     // Render the template
     template.render(&target_dir, pkg_manager, &package_name, alpha, mobile)?;
 
+    // Install dependencies
+
+    if install_deps {
+        if pkg_manager.is_node() {
+            if cfg!(target_os = "windows") {
+                let install_cmd_output = Command::new("cmd")
+                    .args(&["/C", pkg_manager.install_cmd().unwrap()])
+                    .spawn()
+                    .unwrap()
+                    .wait_with_output()
+                    .unwrap();
```

Comment:
it is not recommended to use `.spawn()` then call `.wait_with_output()` directly after it, instead you should just call `.output()` which will have the same effect but more idiomatic. 

But in our case we don't need the output, we just need the exist status so you should just call `.status()`

```suggestion
                let status = Command::new("cmd")
                    .args(&["/C", pkg_manager.install_cmd().unwrap()])
                    .status()
                    .unwrap();
```

We also don't want to unwrap here at all, since this will cause a panic and we need to make sure that the instructions are printed as I mentioned above, so here we instead need to do:
```suggestion
                let success = Command::new("cmd")
                    .args(&["/C", pkg_manager.install_cmd().unwrap()])
                    .status()
                    .map(|s| s.success())
                    .unwrap_or(false);
                
                if !success {
                	// print a warining
                }
```